src/SprykerEco/Shared/AfterPay/Transfer/after-pay.transfer.xml

```diff
@@ -280,64 +274,64 @@
+        <property name="availablePaymentMethodNames" type="array" singular="availablePaymentMethodName"/>
+        <property name="riskCheckCode" type="string"/>
+        <property name="customerNumber" type="string"/>
+        <property name="checkoutId" type="string"/>
+        <property name="quoteHash" type="string"/>
+        <property name="outcome" type="string"/>
+        <property name="riskCheckMessages" type="AfterPayRiskCheckMessage[]" singular="riskCheckMessage"/>
     </transfer>
 
     <transfer name="AfterPayCall">
-        <property name="shippingAddress" type="Address" />
-        <property name="billingAddress" type="Address" />
-        <property name="orderReference" type="string" />
-        <property name="idSalesOrder" type="int" />
-        <property name="paymentMethod" type="string" />
-        <property name="email" type="string" />
-        <property name="items" type="Item[]" singular="Item"/>
-        <property name="totals" type="Totals" />
-        <property name="expenses" type="Expense[]" />
+        <property name="shippingAddress" type="Address"/>
+        <property name="billingAddress" type="Address"/>
+        <property name="orderReference" type="string"/>
+        <property name="idSalesOrder" type="int"/>
+        <property name="paymentMethod" type="string"/>
+        <property name="email" type="string"/>
+        <property name="items" type="Item[]" singular="item"/>
+        <property name="totals" type="Totals"/>
+        <property name="expenses" type="Expense[]" singular="expense"/>
         <property name="payments" type="Payment[]" singular="payment"/>
-        <property name="checkoutId" type="string" />
-        <property name="currency" type="string" />
+        <property name="checkoutId" type="string"/>
+        <property name="currency" type="string"/>
     </transfer>
 
     <transfer name="Quote">
-        <property name="afterPayAvailablePaymentMethods" type="AfterPayAvailablePaymentMethods" />
-        <property name="afterPayPaymentReservationId" type="string" />
+        <property name="afterPayAvailablePaymentMethods" type="AfterPayAvailablePaymentMethods"/>
+        <property name="afterPayPaymentReservationId" type="string"/>
     </transfer>
 
     <transfer name="Payment">
         <property name="afterPayInvoice" type="AfterPayPayment"/>
-        <property name="afterPayCheckoutId" type="string" />
-        <property name="afterPayCustomerNumber" type="string" />
-        <property name="afterPayPaymentMethodAvailability" type="bool" />
+        <property name="afterPayCheckoutId" type="string"/>
+        <property name="afterPayCustomerNumber" type="string"/>
+        <property name="afterPayPaymentMethodAvailability" type="bool"/>
     </transfer>
 
     <transfer name="Order">
-        <property name="AfterPayPayment" type="AfterPayPayment"/>
+        <property name="afterPayPayment" type="AfterPayPayment"/>
```

Comment:
Please double check that this isnt a BC break (that the method generation is now the same as before).

Reply:
Yes, I did triple check)
So, this code allow us to don't worry about letter register in `Generated\Shared\Transfer\OrderTransfer`:

    ```
protected $transferPropertyNameMap = [
        'after_pay_payment' => 'afterPayPayment',
        'afterPayPayment' => 'afterPayPayment',
        'AfterPayPayment' => 'afterPayPayment', 
```